test/jasmine/tests/hover_label_test.js

```diff
@@ -123,7 +125,8 @@ describe('hover info', function() {
 
             expect(d3.selectAll('g.axistext').size()).toEqual(0);
             expect(d3.selectAll('g.hovertext').size()).toEqual(1);
-            expect(d3.selectAll('g.hovertext').select('text').html()).toEqual('hover text');
+            expect(d3.selectAll('g.hovertext').select('text').html())
+                .toEqual('hover text  with spaces  not newlines');
```

Comment:
None of the image tests in https://github.com/plotly/plotly.js/pull/1332/commits/21059303ee919812e4b2e281df5f71f7c77cf7be actually changed results with the fix there, but they would all have resulted in invisible text in IE before the fix. But this test confirms that we do sanitize newlines for IE compatibility.